source/PcAux/R/exportedHelperFunctions.R

```diff
@@ -297,7 +297,6 @@ calcTime <- function(pcAuxData, what) {
 
 ## Write machine status to a text files 
 writeStatus <- function(pcAuxData, outName, what) {
-    fileName <- file(outName)
```

Comment:
Explicitly creating a connection here only seems to cause problems. The capture.output function should create and close any necessary connections.